cfme/v2v/migrations.py

```diff
@@ -414,7 +414,8 @@ class vms(View):  # noqa
 
         @property
         def is_displayed(self):
-            return self.filter_by_dropdown.is_displayed
+            return (self.filter_by_dropdown.is_displayed and
+                 (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0))
```

Comment:
Where are you seeing this behavior?

ensure_page_safe should be handling this before navigation is attempted.

Reply:
it happens in modal.